Engine/include/Texture/ImageGenerator.h

```diff
@@ -1,4 +1,5 @@
 #pragma once
+#include <cstdint>
```

Comment:
While on I think all the most popular C++ stdlib implementations `cstdint` header also exports symbols in the global namespace like `stdlib.h`, it's not required to.
I see that only the latter header is used in the repo, so this should also use `stdlib.h`.
One little problem with it is annoying clang-tidy "deprecated" warnings, but typing out std:: everywhere is even worse.